uber/templates/fields/attendee.html

```diff
@@ -789,16 +781,13 @@ <h4>{{ tier.name }}</h4>
         if ($.field('staffing')) {
             $.field('staffing').on('click', staffingClicked);
         }
-        noCellphoneClicked();
-        if ($.field('no_cellphone')) {
-            $.field('no_cellphone').on('click', noCellphoneClicked);
-        }
         {% if admin_area %}
             if($.field('staffing')) {
                 toggleStaffingAdminRows();
                 $.field('staffing').on('click', toggleStaffingAdminRows);
             }
         {% endif %}
+        staffingClicked();
```

Comment:
Oh yea, this fixes a bug where non-admin pages would always show staffing rows (e.g. 'where do you want to volunteer?') by default.